using_images/other_images/jenkins.adoc

```diff
@@ -169,51 +162,109 @@ To create a new Jenkins application using standard Jenkins authentication:
 ----
 $ oc new-app -e \
     JENKINS_PASSWORD=<password> \
-    openshift/jenkins-1-centos7
+    openshift/jenkins-2-centos7
 ----
 
 [[jenkins-environment-variables]]
 === Environment Variables
 
 The Jenkins server can be configured with the following environment variables:
 
-.Jenkins Environment Variables
-[cols="4a,6a",options="header"]
-|===
+* `JENKINS_PASSWORD` (default: `password`)
++
+The password for the `admin` user when using standard Jenkins authentication.
+Not applicable when using {product-title} OAuth authentication.
 
-|Variable name |Description
+* `OPENSHIFT_ENABLE_OAUTH` (default: `false`)
++
+Determines whether the OpenShift Login plug-in manages authentication when
+logging into Jenkins. To enable, set to `true`.
 
-|`*JENKINS_PASSWORD*`
-|The password for the `*admin*` user when using standard Jenkins authentication.
-Not applicable when using {product-title} OAuth authentication.
+* `CONTAINER_HEAP_PERCENT` (default: `0.5`, i.e. 50%)
++
+Specifies the percentage of the container memory limit allocated to the Jenkins
+JVM maximum heap size (-Xmx).
+
+* `JENKINS_MAX_HEAP_UPPER_BOUND_MB`
++
+If set, specifies a static upper bound in MiB for the Jenkins JVM maximum heap
+size (-Xmx).  The effective setting used is the minimum of
+`CONTAINER_HEAP_PERCENT` and `JENKINS_MAX_HEAP_UPPER_BOUND_MB`. Example setting:
+`512`.
+
+* `JENKINS_JAVA_OPTIONS`
++
+Specifies additional options for the Jenkins JVM which may be overridden.
 
-|`*OPENSHIFT_ENABLE_OAUTH*`
-|Determines whether the OpenShift Login plug-in manages authentication when
-logging into Jenkins. Enabled when set to any non-empty value other than
-"false".
+* `JENKINS_JAVA_OVERRIDES`
++
+Specifies additional options for the Jenkins JVM which may not be overridden.
+
+* `JENKINS_OPTS`
++
+Specifies additional Jenkins arguments.
 
-|`*OPENSHIFT_PERMISSIONS_POLL_INTERVAL*`
-|Specifies in seconds how often the OpenShift Login plug-in polls {product-title}
-for the permissions associated with each user defined in Jenkins.
+* `CONTAINER_CORE_LIMIT`
++
+If set, specifies an integer number of cores used for sizing numbers of internal
+JVM threads.  Example setting: `2`.
+
+* `OPENSHIFT_JENKINS_JVM_ARCH`
++
+Set to `x86_64` or `i386` to override the JVM used to host Jenkins.  For memory
+efficiency, by default the Jenkins image dynamically uses a 32-bit JVM if
+running in a container with a memory limit under 2GiB.
 
-|`*OVERRIDE_PV_CONFIG_WITH_IMAGE_CONFIG*`
-|When running this image with an {product-title} persistent volume for the Jenkins
+* `JAVA_MAX_HEAP_PARAM`
++
+If set, entirely overrides the Jenkins JVM maximum heap size (-Xmx).  Example
+setting: `-Xmx512m`.
```

Comment:
Need to discuss how this interacts with JENKINS_JAVA_OPTIONS and JENKINS_JAVA_OVERRIDES.  And CONTAINER_HEAP_PERCENT.